sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetIOSuite.scala

```diff
@@ -107,7 +107,7 @@ class ParquetIOSuiteBase extends QueryTest with ParquetTest {
         // Parquet doesn't allow column names with spaces, have to add an alias here
         .select($"_1" cast decimal as "dec")
 
-    for ((precision, scale) <- Seq((5, 2), (1, 0), (1, 1), (18, 10), (18, 17))) {
+    for ((precision, scale) <- Seq((5, 2), (1, 0), (1, 1), (18, 10), (18, 17), (19, 0), (38, 37))) {
```

Comment:
Why should it fail? Decimals with large precisions are available no matter `followParquetFormatSpec` is true or false, right? For your local test failure, I guess a clean would probably solve the problem.